report-generator/.cqfd/docker/Dockerfile

```diff
@@ -10,10 +10,13 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	asciidoc \
 	bash \
 	locales \
+	python3-pip \
```

Comment:
Don't you prefer installing python3-matplotlib directly?
We don't use pip afterwards.